packages/express-backend/backend.js

```diff
@@ -84,6 +85,21 @@ app.get("/playlists/:id", async (req, res) => {
   }
 });
 
+app.get("/playlists/:id/comments", async (req, res) => {
+  try {
+    const playlistId = req.params["id"];
+    const result = await commentsServices.getAllCommentsByPlaylistId(playlistId);
```

Comment:
This doesn't really follow our data structure. Playlists should have an array of Comments. Then, you would want to search for playlist and retrieve the comments of that playlist. This would be how likes should work to. Reference my AddSong if needed. This should be a quick change but let me know if you need any help.

Reply:
Yes I agree with this as well, each Playlist should have a comment_list or something of the sort, which is an array of comments associated with that specific playlist